### ISSUE TITLE
Fails with PrependTo::normal message

### ISSUE THREAD

**Issue by bbgpdfrey** [szhorvat/SEUploader#10]
I recently installed SEUploader using the recommended script but receive the error message, 

Nonatomic expression expected at position 1 in PrependTo[CurrentValue[SEUploader`Private`pnb,{TaggingRules,ImageUploadHistory}], {... , http://i.stack.imgur.com/wVsMh.png}]. >> 

whenever I try to use it.  I have Mathematica 10.0.2.0 running on Windows 8.1 (64 bit) and have tried both Internet Explorer and FireFox with the same result.

**Comment by szhorvat** [comment]
Did you try the latest version?  http://meta.mathematica.stackexchange.com/a/32/12

This repository has an old version that broke after some changes to SE.

**Comment by bbgpdfrey** [comment]
I believe that I did, although I could load it again to verify that I have the most current version.  If I do, is it necessary for me first to uninstall the version now on  my computer?  By the way, someone else noted the same problem in a Comment dated 18 July 14, which appears to be later than the Update.  Thanks.

**Comment by szhorvat** [comment]
Yes, it is necessary to uninstall the old version first.  Please follow the instructions [here](https://github.com/halirutan/SEUploaderApplication) and let me know if there is still a problem.

**Comment by bbgpdfrey** [comment]
Followed instructions precisely.  Same error message as before when I try to use upload a graphic.

**Comment by szhorvat** [comment]
Shall we continue in private mail?  You know how to find my email address.
